catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableFileAlterationListener.java

```diff
@@ -123,4 +122,24 @@ void checkFiles() {
     }
     completedFiles.stream().forEach(file -> fileMap.remove(file));
   }
+
+  static class Pair<K, V> {
```

Comment:
❓ How about using https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/tuple/package-summary.html instead?

Reply:
It seemed simple enough to use an inner class at the time but I updated it to use the apache class.